src/components/ScrollBox/ScrollBox.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+export interface ScrollBoxProps {
+  color?: string;
+  radius?: string;
+  padding?: string;
+  height?: string;
+  thumbheight?: string;
+}
```

Comment:
```suggestion
export interface ScrollBoxProps {
  color: string;
  radius: string;
  padding: string;
  height: string;
  thumbheight: string;
}
```

변경 하신 후 사용하시는 곳에서
```typescript
Partial<ScrollBoxProps>
```

로 사용하실 수 있습니다 ㅎㅎ